src/main/java/io/kestra/plugin/singer/taps/PipelinewiseSqlServer.java

```diff
@@ -0,0 +1,185 @@
+package io.kestra.plugin.singer.taps;
+
+import com.google.common.collect.ImmutableMap;
+import io.kestra.core.exceptions.IllegalVariableEvaluationException;
+import io.kestra.core.models.annotations.Example;
+import io.kestra.core.models.annotations.Plugin;
+import io.kestra.core.models.annotations.PluginProperty;
+import io.kestra.core.models.tasks.RunnableTask;
+import io.kestra.core.runners.RunContext;
+import io.kestra.plugin.singer.models.Feature;
+import io.swagger.v3.oas.annotations.media.Schema;
+import lombok.*;
+import lombok.experimental.SuperBuilder;
+
+import javax.validation.constraints.NotEmpty;
+import javax.validation.constraints.NotNull;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+
+@SuperBuilder
+@ToString
+@EqualsAndHashCode
+@Getter
+@NoArgsConstructor
+@Schema(
+    title = "A Singer tap to fetch data from a Microsoft SQL Server database.",
+    description = "Full documentation can be found [here](https://github.com/BuzzCutNorman/tap-mssql/blob/main/README.md)"
+)
+@Plugin(
+    examples = {
+        @Example(
+            code = {
+                "host: 127.0.0.1",
+                "username: SA",
+                "password: sqlserver_passwd",
+                "port: 57037",
+                "filterDbs: dbo",
+                "streamsConfigurations:",
+                "  - stream: Categories",
+                "    replicationMethod: INCREMENTAL",
+                "    replicationKeys: CategoryID",
+                "    selected: true",
+                "  - propertiesPattern:",
+                "      - Description",
+                "    selected: false",
+            }
+        )
+    }
+)
+public class PipelinewiseSqlServer extends AbstractPythonTap implements RunnableTask<AbstractPythonTap.Output> {
+    @NotNull
+    @NotEmpty
+    @Schema(
+        title = "The database hostname"
+    )
+    @PluginProperty(dynamic = true)
+    private String host;
+
+    @NotNull
+    @NotEmpty
+    @Schema(
+        title = "The database name"
+    )
+    @PluginProperty(dynamic = true)
+    private String database;
+
+    @NotNull
+    @NotEmpty
+    @Schema(
+        title = "The database port"
+    )
+    @PluginProperty
+    private Integer port;
+
+    @NotNull
+    @NotEmpty
+    @Schema(
+        title = "The database user"
+    )
+    @PluginProperty(dynamic = true)
+    private String username;
+
+    @NotNull
+    @NotEmpty
+    @Schema(
+        title = "The database user's password"
+    )
+    @PluginProperty(dynamic = true)
+    private String password;
+
+    @Schema(
+        title = "Schema to extract tables only from particular schemas and to improve data extraction performance"
+    )
+    @PluginProperty(dynamic = true)
+    private String filterDbs;
```

Comment:
Should be a list like for the other taps:

```java
@Schema(
        title = "The list of schemas to extract tables only from particular schemas and to improve data extraction performance"
    )
    @PluginProperty(dynamic = true)
    private List<String> filterDbs;
```

Reply:
The doc was not saying it is a comma separated config so I did not put it but after looking the target code it looks to do so, ty